docs/model-dev-guide/best-practices/_index.rst

```diff
@@ -1,75 +0,0 @@
-################
- Best Practices
-################
-
-***************************************
- General Tips for the Trial Definition
-***************************************
-
-Do:
-
--  Use framework abstractions to implement learning rate scheduling instead of directly changing the
-   learning rate. See `tf.keras.optimizers.schedules.LearningRateSchedule
-   <https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/LearningRateSchedule>`__
-   and :class:`determined.pytorch.LRScheduler` as examples.
-
--  For code that needs to download artifacts (e.g., data, configurations, pretrained weights),
-   download to a `tempfile.TemporaryDirectory <https://docs.python.org/3/library/tempfile.html>`__
-   unique to the Python process. This will avoid race conditions when using distributed training, in
-   which Determined executes multiple Python processes in the same task container.
-
-.. include:: ../../_shared/note-dtrain-learn-more.txt
-
-Do not use instance attributes on a trial class to save any state over time (e.g., storing metric
-history in a ``self`` attribute). The ``Trial`` instance will only save and restore model weights
-and optimizer state over time; ``self`` attributes may be reset to their initial state at any time
-if the Determined cluster reschedules the trial to another task container.
-
-**********************************
- Separate Configuration from Code
```

Comment:
this is from a time when we thought people would love the experiment config, but people routinely complain that they'd rather just run their code like they normally do.